packages/core/src/Avatar/Avatar.styles.tsx

```diff
@@ -49,5 +49,5 @@ export const { staticClasses, useClasses } = createClasses("HvAvatar", {
     zIndex: 1,
   },
   circular: { borderRadius: theme.radii.full },
-  square: {},
+  square: { borderRadius: theme.radii.round },
```

Comment:
this makes no sense to me... 😅

We could instead align with the MUI API and add the `rounded` variant. Our `variant` API is already a subset of MUI's, with only the `rounded` missing.
It probably would be better to align with `HvRadius`, but now it'd just add more confusion - maybe we can add a `// TODO: align with `HvRadius` if it makes sense 

Alternatively, this should be at least in the `ds5` overrides, not here in the "UI Kit default styles"